docs/Using Price Feeds/l2-sequencer-feeds/l2-sequencer-flag.md

```diff
@@ -0,0 +1,166 @@
+
+1. The `optimismValidator` contract calls the `sendMessage` function in the `L1CrossDomainMessenger` contract. This message contains instructions to call the `updateStatus(bool status, uint64 timestamp)` function in the sequencer uptime feed deployed on the L2 network.
+
+1. The `L1CrossDomainMessenger` contract calls the `enqueue` function to enqueue a new message to the `CanonicalTransactionChain`.
+
+1. The `Sequencer` processes the transaction enqueued in the `CanonicalTransactionChain` contract to send it to the L2 contract.
+
+**On the L2 network:**
+
+1. The `Sequencer` posts the message to the `L2CrossDomainMessenger` contract.
+
+1. The `L2CrossDomainMessenger` contract relays the message to the `OptimismSequencerUptimeFeed` contract.
+
+1. The message relayed by the `L2CrossDomainMessenger` contains instructions to call `updateStatus` in the `OptimismSequencerUptimeFeed` contract.  
+
+1. Consumers can then read from the `AggregatorProxy` contract, which fetches the latest round data from the `OptimismSequencerUptimeFeed` contract.
+
+### Handling outages on Optimism and Metis
+
+There are two scenarios that need to be handled whenever the Sequencer goes down:
+
+- [L1 to L2 transactions](#l1-to-l2-transactions)
+- [Direct Sequencer Transactions](#direct-sequencer-transactions)
+
+#### L1 to L2 transactions
+
+If the sequencer is down, messages cannot be transmitted from L1 to L2 and **no L2 transactions are executed**. Instead, messages are enqueued in the `CanonicalTransactionChain` on L1 and only processed in the order they arrived later when the sequencer comes back up. As long as the message from the validator on L1 is already enqueued in the `CTC`, the flag on the sequencer uptime feed on L2 will be guaranteed to be flipped prior to any subsequent transactions. The transaction that flips the flag on the uptime feed will be executed before transactions that were enqueued after it. This is further explained in the diagrams below.
+
+When the Sequencer is down, all L2 transactions sent from the L1 network wait in the pending queue.
+
+1. **TX3** contains Chainlink’s transaction to set the status of the sequencer as being down on L2.
+1. **TX4** is a transaction made by a consumer that is dependent on the sequencer status.
+
+![L2 Sequencer Feed Diagram](/images/data-feed/seq-down-1.png)
+
+After the sequencer comes back up, it moves moves all transactions in the pending queue to the processed queue.
+
+1. Transactions are processed in the order they arrived so **TX3** is processed before **TX4**.
+1. Because **TX3** happens before **TX4**, **TX4** will read the status of the Sequencer as being down and responds accordingly.
+
+![L2 Sequencer Feed Diagram](/images/data-feed/seq-down-2.png)
+
+#### Direct Sequencer Transactions
```

Comment:
I don't think this section is true any longer as the feed only updates once every 24 hours if the status has changed.  Can we remove it for now?

Reply:
Removed